src/openforms/forms/api/serializers/form.py

```diff
@@ -55,6 +55,33 @@ class FormLiteralsSerializer(serializers.Serializer):
     confirm_text = ButtonTextSerializer(raw_field="confirm_text", required=False)
 
 
+class FormAuthenticationBackendSerializer(serializers.ModelSerializer):
+    options = serializers.JSONField(
+        label=_("authentication backend options"), allow_null=True, required=False
+    )
+
+    class Meta:
+        model = FormAuthenticationBackend
+        fields = [
+            "backend",
+            "options",
+        ]
```

Comment:
can you stick to the pattern of using tuples for these sort of declarations? tuples are immutable and convey the nature of this configuration better